### PR TITLE
Fix JHU's fips-population source file

### PR DESCRIPTION
**Pull request by dshemetov**
* use geocode utility source file and remove old static source file
* update tests
* update validation notebook to verify prop signals in NYC

### CODE REVIEW

**Comment by krivard** [comment]
It could've gone either way, but I switched the base to `deploy-jhu` because this PR primarily concerns the JHU indicator. Once we merge it, we'll propagate the changes to `main`.

File: testing_utils/indicator_validation.ipynb [hunk 1]
```diff
@@ -172,7 +172,130 @@
...
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Total difference:  2.3797075421327918e-12\n"
+     ]
+    },
+    {
+     "output_type": "display_data",
+     "data": {
+      "text/plain": "<Figure size 1152x432 with 1 Axes>",
```
Comment (sarah-colq) [inline testing_utils/indicator_validation.ipynb:241]: Should we be able to see the remote line or are they identical here?  I cannot see the remote line on any of the new charts.

**Review by sarah-colq — CHANGES_REQUESTED** [review]
Please check the charts in the python notebook.  Otherwise, looks good! :)

**Comment by dshemetov** [comment]
That notebook isn't really supposed to be tracked. It was meant to be a template, but I kept working in the template and committed changes. I've reverted it back to its template version and renamed it accordingly.

**Review by krivard — APPROVED** [review]

**Pull request merged**